src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -1738,9 +1738,9 @@ private async Task<int> EnumerateItemsFromStandardFolderAsync(string path, Cance
 
 						await OrderFilesAndFoldersAsync();
 						await ApplyFilesAndFoldersChangesAsync();
+						await dispatcherQueue.EnqueueOrInvokeAsync(CheckForSolutionFile, Microsoft.UI.Dispatching.DispatcherQueuePriority.Low);
```

Comment:
CheckForSolutionFile is not related to GetDesktopIniFileData and CheckForBackgroundImage, so I separated the queuing.